src/flows/components/FlowsIndex.tsx

```diff
@@ -61,32 +107,62 @@ const FlowsIndex = () => {
       titleTag={pageTitleSuffixer([PROJECT_NAME_PLURAL])}
       testID="flows-index"
     >
-      <PageHeader fullWidth={false}>
+      <PageHeader
+        fullWidth={false}
+        className={`${showButtonMode && 'withButtonHeader marginHeader'}`}
```

Comment:
i didn't realize you created both of these classes. merge their definitions and get rid of one